Dockerfile.local

```diff
@@ -1,6 +1,13 @@
 FROM node:15.14
 COPY . .
 ENV NODE_ENV=development
+# --- NETFREE CERT INTSALL ---
```

Comment:
interesting. what does it do?

Reply:
Its just for me , I will delete it .
Installs a specific security certificate, otherwise NPM fails, because I have a filter on the network of the Internet at home

Reply:
nice..  maybe you can add a StackOverflow manual on "how to pass the network filter in greenpress"